metadata_ui.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var Mocha = require('mocha'),
```

Comment:
Okay, last structural review/comment here:

If you could create this module in such a way that it "extended" a passed in instance of mocha then you wouldn't need to make mocha a dependency at all. For instance:

metadata_ui.js:
```
module.exports = function(mocha) {
  var Mocha = mocha; 
  var Suite = mocha.Suite; // I have no idea if these are even exported..
  var Test = mocha.Test;  

  return function(suite) {
     ... all the rest ...
  };
};
```

Then this could be reusable in this type of form, say..
runner.js

```
var mocha = require('mocha');
var metadata_ui = require('metamocha')(mocha);
```

As I commented above, I'm not sure mocha exports things this way - so if the idea doesn't work out, don't worry we can work on this next week.